### ISSUE TITLE
Show touristical information=map's?

### ISSUE THREAD

**Issue by Hufkratzer** [lonvia/waymarked-trails-site#264]
Is there a special reason why you show https://wiki.openstreetmap.org/wiki/Tag:information=guidepost but not https://wiki.openstreetmap.org/wiki/Tag:information=map? I think they have a very similar purpose; and both are tagged with hiking=yes, bicycle=yes, mtb=yes, horse=yes or ski=yes.

Wanderreitkarte.de shows them, example:
http://www.wanderreitkarte.de/index.php?lon=16.1471&lat=50.6535&zoom=18&lang=en
http://www.openstreetmap.org/node/3707010538

If you are going to modernize guideposts/signposts now (https://github.com/lonvia/waymarked-trails-site/issues/262) perhaps this would be an opportunity to include these touristical maps too.

**Comment by lonvia** [comment]
Yes, that can be done. Looking at taginfo, there are about 40k maps tagged. About half of them have a hiking/bicycle/mtb/horse/ski=yes tag. Those are the interesting ones.

For icons: JOSM uses a simple 'i' in a circle in rectangle for information=map. The main map only renders a generic 'i'. There is nothing suitable in the maki icon set.

**Comment by Hufkratzer** [comment]
I think, for the information map icon something like in wanderreitkarte.de would be better than an 'i'.

Do you want/need to have the icons in https://github.com/mapbox/maki? Disadvantage; They want every icon in two sizes. Advantage: iD could use them too. But the guidepost icon also isn't there yet.

**Comment by lonvia** [comment]
I'm not particularly attached to maki. It's simply one of the few public domain map icon sets I'm aware of.

The SJJB icons contain icons for guideposts and map, that would be an option. See http://www.sjjb.co.uk/mapicons/contactsheet

**Comment by alpter** [comment]
Hello... RE: icons ,  I use this .... [https://thenounproject.com](url)  Would be great to have difference between information=map   and  information=guidepost   -  Some guidepost example... here 
[https://thenounproject.com/search/?q=guidepost](guidepost)
